### DISCUSSION TITLE
stuck with commit history in 1970

### DISCUSSION THREAD

**Discussion by behind24proxies** [community/community#33255] [Other Feature Feedback, Questions, & Ideas]
hey there , this is my profile 
https://github.com/behind24proxies
a couple of weeks ago, I tried making a fake commit by setting the date to 1 jan 1970 , it surprisingly worked and I could see a shiny green box in 1970.
well, now I don't want it anymore, I tried deleting the repo in hopes of getting rid of if , but it did not work.
any suggestions? 
![image](https://user-images.githubusercontent.com/96356684/190628913-a5d9f7ea-3df1-4a42-82be-f7bd8b5ad21f.png)

**Comment by strong-728** [comment]
Hello, behined24proxies!
It is wrong idea for fake commit history on github. 
I think if all new github users do such way for itself, our github playground may be messy because of many scammers.
Dont discuss about this here.
Thanks,

**Reply by airtower-luna** [reply]
Please don't make misleading statements, there's no harm here. Scammers and spammers are a problem, but not people playing with their own repository histories.

It's good if people play with the tools they use and figure out what they can do. Some even [make art with their contribution graph](https://github.com/gelstudios/gitfiti)! :grin: And I've seen other creative uses, like tracking changes in laws and setting the author dates to the dates of the actual changes. Which I actually heard about because they asked on the forum about trouble with dates _before_ 1970, but I can't find the link right now. :sweat_smile:

**Reply by strong-728** [reply]
what do you mean?
you mean you agree with fake git? really disappointed about you and all user must be true in their account and it is not a toy anymore on our developing business.

**Reply by strong-728** [reply]
yeah, dont know how to do it but there will be a way absolutely because of he already wrote on github history.
with the project he used, we can try all together but this is not good work.
thanks again.

**Comment by riley-dickerson** [comment]
Oh, great.
I also have the same issue from my funny mistake, fake years are not displayed in my profile.
But others can see it, and I do not want it.
If I delete the repository, it's fake year will be removed automatically after 90 days?
Without helping from the github support team.
How do you think?
